src/config.rs

```diff
@@ -104,6 +115,8 @@ pub struct Config {
 pub enum Error {
     #[error("invalid env var {0}={1}")]
     EnvVar(String, String),
+    #[error("error occurred: {0}")]
```

Comment:
This gives a confusing error: `Error: error occurred: invalid type: integer `1`, expected a map`; without context its not clear what the issue is